package.json

```diff
@@ -25,7 +29,8 @@
   },
   "homepage": "https://github.com/fraserxu/electron-pdf",
   "devDependencies": {
-    "standard": "^6.0.6"
+    "jasmine": "^2.5.2",
```

Comment:
I have ava configured on [another branch (in the future)](https://github.com/codecounselor/electron-export/compare/issue-110-multiple-urls...codecounselor:jasmine-to-ava?expand=1) but what I found is that any test which requires Electron can't be run with Ava.  If you know a way around this please tell me.

I am thinking there can just be a small set of integration tests that runs in Jasmine (for now), like this:
```
"test": "npm run lint && ava",
"integration-test": "electron node_modules/.bin/jasmine"
```